src/main/webapp/courseLoader.js

```diff
@@ -238,64 +237,41 @@
   async function getRatingPropertiesToStore() {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
+
     const ratingProperties = {
-      courseName: urlParams.get("course-name"),
-      term: urlParams.get("term"),
-      profName: urlParams.get("prof-name"),
-      units: urlParams.get("num-units"),
-      schoolName: urlParams.get("school-name"),
+      courseKey: urlParams.get("course-key"),
+      termKey: urlParams.get("term-key"),
       termInput: document.getElementById("term-input").value,
       profInput: document.getElementById("prof-input").value,
       ratingTerm: document.getElementById("rating-term").value,
       ratingProf: document.getElementById("rating-prof").value,
       hours: document.getElementById("hours").value,
       difficulty: document.getElementById("difficulty").value,
+      ID: await verify(),
```

Comment:
Nit: Acronyms are still typed in lowercase by convention (so this should just be "id")